.appveyor.yml

```diff
@@ -345,9 +358,9 @@ test_script:
   - cmd: md __testhome__
   - sh: mkdir __testhome__
   - cd __testhome__
-    # run test selecion (--traverse-namespace needed from Python 3.8 onwards)
-  - cmd: python -W error::DeprecationWarning:^datalad -m nose --traverse-namespace -s -v -A "not (turtle)" --with-cov --cover-package datalad %DTS%
-  - sh: PATH=$PWD/../tools/coverage-bin:$PATH python -W error::DeprecationWarning:^datalad -m nose --traverse-namespace -s -v -A "not (turtle)" --with-cov --cover-package datalad ${DTS}
+    # run test selection
+  - cmd: python -m pytest -c ../tox.ini -s -v -m "not (turtle)" --cov=datalad -k test_tests_utils --pyargs %DTS%
```

Comment:
we need to come up with a "scaleable" and convenient way avoiding duplication (between `cmd` and `sh` here + travis) of whitelisting test files. E.g. we can add quite a few right away, e.g. `datalad/tests/test_version.py` which seems to work without any changes . I am also not sure if `-k` is the best way since then e.g. `test_base` might match multiple.

Reply:
another ugly one is to rename all `test_*.py` into `_test_*.py` and then unrename subsequently... or we could inject having `raise SkipTest` in them but also don't like that.

Reply:
Another option would be to delete all but one matrix element from the CI config and list the currently-converted test files in that element.

Reply:
So should I add a matrix entry for the restricted `TESTS_TO_PERFORM` or modify one of the existing instances?  If the latter, which one?

Reply:
latter -- the one for `datalad`, and comment out the other one for now.